### DISCUSSION TITLE
Chrome notice that the Finicky.zip may be dangerous

### DISCUSSION THREAD

**Discussion by adrianoresende** [johnste/finicky#244] [General]
My question is should I install it? 🤔 

<img width="413" alt="image" src="https://user-images.githubusercontent.com/798755/153919489-826ecbc8-2fb0-46e8-a09a-dc404b7a59e2.png">

**Comment by johnste** [comment]
Hey. I'm not sure why Chrome would warn about the file, but I did see another warning the other week where my Mac warned about starting Finicky. https://github.com/johnste/finicky/issues/240

I find this all to be pretty odd, the release is unchanged from March 2021 when it was released. I ran the zip-file and the app separately through an online virus detector and the tool could not find anything suspicious in it either.
- https://www.virustotal.com/gui/file/da572a168b670d3271e449ac83d55f1d642aa73695ef50da0c022c0f19da925c
- https://www.virustotal.com/gui/file/e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855

I got an e-mail late January from apple telling me I had revoked my development certificate (I didn't revoke it), I wonder if it could be related to these issues

> You have revoked your certificate, so it is no longer valid.
